source/write/transactions.txt

```diff
@@ -0,0 +1,170 @@
+.. _pymongo-write-transactions:
+
+============
+Transactions
+============
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: ACID, write, consistency, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-long+} to perform
+**transactions**. :manual:`Transactions </core/transactions/>` allow
```

Comment:
S: based on the [style guidelines](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/) on links (especially [this section](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear) on clarifying where you're linking to), I think you can remove the inline links from this paragraph. Since you include links to the Server manual pages at the end of this guide, I'd just keep them there and bold the terms in this paragraph